src/App.js

```diff
@@ -1,27 +1,10 @@
-import logo from './logo.svg';
 import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
```

Comment:
- [ ] Please as it is a requirement for this project, `App.js` should be `class-based`. Please kindly fix that.

Reply:
Hello @Mihndim2020,
Aww, I totally forget that file since it was also a part of boilerplate :) Thank you for remembering it and the review. I'm fixing it right now :)